playbooks/deploy.yaml

```
@@ -56,7 +60,13 @@
       - { type: host, database: all, user: all, address: '127.0.0.1/32', auth_method: md5 }
       - { type: host, database: all, user: all, address: '::1/128', auth_method: md5 }
   roles:
-    - certificates
+    - role: certificates
+      when: "certificate_source | default('default') == 'default'"
```

```suggestion
      when: "certificate_source == 'default'"
```
(this should work with the latest changes)